core/types/calls.cc

```diff
@@ -914,6 +914,20 @@ DispatchResult dispatchCallSymbol(const GlobalState &gs, const DispatchArgs &arg
                     }
                 }
             }
+
+            auto attachedClass = symbol.data(gs)->attachedClass(gs);
+            if (attachedClass.exists()) {
+                for (auto mixin : attachedClass.data(gs)->mixins()) {
+                    if (mixin.data(gs)->findMethod(gs, targetName).exists()) {
+                        e.addErrorNote("The method `{}` exists as an instance method in module `{}`",
+                                       targetName.show(gs), mixin.show(gs));
+                        e.addErrorNote(
+                            "Consider changing `include {}` to `extend {}` to make it available as a class method",
+                            mixin.show(gs), mixin.show(gs));
```

Comment:
We aim to only have one `addErrorNote` call per error.

Let's combine these errors into one string (probably, with newlines between the two sentences) and only call `addErrorNote` once.

You can see an example of that being done here that you should aim to copy

https://github.com/sorbet/sorbet/blob/master/resolver/type_syntax/type_syntax.cc#L1275-L1277